src/components/one-time-donation/steps/FirstStep.tsx

```diff
@@ -124,6 +136,32 @@ export default function FirstStep() {
                 .concat({ label: 'Other', value: 'other' }) || []
             }
           />
+          <Collapse in={amount.value === 'other'} timeout="auto">
+            <Grid
+              style={
+                !mobile
+                  ? {
+                      float: 'right',
+                      marginTop: theme.spacing(-10),
+                      width: 325,
```

Comment:
Using hardcoded `width: 325` might be too restrictive for the responsive styles.

Maybe using `100%` will deliver the same results, can you confirm